src/test/unit/lib/model/OutboundTransfersModel.test.js

```diff
@@ -215,6 +216,98 @@ describe('outboundModel', () => {
     });
 
 
+    test('uses quote response transfer amount for transfer prepare', async () => {
+        defaultEnv.AUTO_ACCEPT_PARTY = 'true';
+        defaultEnv.AUTO_ACCEPT_QUOTES = 'true';
+
+        await setConfig(defaultEnv);
+        const conf = getConfig();
+        const cache = new Cache();
+
+        MojaloopRequests.__getParties = jest.fn(() => {
+            cache.emitMessage(JSON.stringify(payeeParty));
+            return Promise.resolve();
+        });
+
+        const dummyTransferAmount = {
+            currency: 'XYZ',
+            amount: '9876543210'
+        };
+
+        MojaloopRequests.__postQuotes = jest.fn(() => {
+            // ensure that the `MojaloopRequests.postQuotes` method has been called with correct arguments
+            // including extension list
+            expect(MojaloopRequests.__postQuotes).toHaveBeenCalledTimes(1);
+            const extensionList = MojaloopRequests.__postQuotes.mock.calls[0][0].extensionList;
+            expect(extensionList).toBeTruthy();
+            expect(extensionList.extension).toBeTruthy();
+            expect(extensionList.extension.length).toBe(2);
+            expect(extensionList.extension[0]).toEqual({ key: 'qkey1', value: 'qvalue1' });
+            expect(extensionList.extension[1]).toEqual({ key: 'qkey2', value: 'qvalue2' });
+
+            // simulate a callback with the quote response
+            // note that we will change the currency and amount of the quote response to ensure
+            // the transfer prepare uses those rather than the originally requested currency.
+
+            // first make sure nobody has changed the quote response data
+            expect(quoteResponse.data.transferAmount).toEqual({
```

Comment:
I suppose you wanted to check the `postQuotes` request arg
```MojaloopRequests.__postQuotes = jest.fn(request => { ... })```
as `quoteResponse` (defined as `const quoteResponse = require('./data/quoteResponse');`) is never changed.

Reply:
I just want to make sure the value in the mock data has not been changed so the test remains valid. if someone changed the mock data to match the overridden amount, the test would no longer be valid but would pass.